requests/models.py

```diff
@@ -447,9 +448,14 @@ def prepare_headers(self, headers):
         self.headers = CaseInsensitiveDict()
         if headers:
             for header in headers.items():
+                name, value = header
+                if value is None:
+                    if name.lower() in urllib3.util.SKIPPABLE_HEADERS:
+                        value = urllib3.SKIP_HEADER
```

Comment:
This is going to fail for older versions of urllib3, right? What do you think about doing something in compat like:

```Python
try:
    import urllib3
    SKIP_HEADER = urllib3.SKIP_HEADER
    SKIPPABLE_HEADERS = urllib3.util.SKIPPABLE_HEADERS
except AttributeError:
    SKIP_HEADER = None
    SKIPPABLE_HEADERS = []
```

Reply:
Also this is giving an attribute error, should be `urllib3.util.SKIP_HEADER`

Reply:
Done and done. Good call @nateprewitt